### PR TITLE
Add missing dependencies to the coq-mi-cho-coq

### PR DESCRIPTION
**Pull request by arvidj**
Add the complete set of dependencies specified in https://gitlab.com/nomadic-labs/mi-cho-coq/blob/master/coq-mi-cho-coq.opam

### CODE REVIEW

File: extra-dev/packages/coq-mi-cho-coq/coq-mi-cho-coq.dev/opam [hunk 1]
```diff
@@ -15,10 +15,17 @@ install: [
   make "install"
 ]
 depends: [
-  "ocaml"
-  "coq" {>= "8.8"}
+  "coq-menhirlib" {>= "20190626"}
   "coq-ott" {>= "0.29"}
+  "coq" {>= "8.8"}
+  "coq-ott"
```
Comment (palmskog) [inline extra-dev/packages/coq-mi-cho-coq/coq-mi-cho-coq.dev/opam]:
```suggestion
```
This dep is already specified just below.

File: extra-dev/packages/coq-mi-cho-coq/coq-mi-cho-coq.dev/opam [hunk 2]
```diff
@@ -15,10 +15,17 @@ install: [
...
+  "ott"
```
Comment (palmskog) [inline extra-dev/packages/coq-mi-cho-coq/coq-mi-cho-coq.dev/opam]:
```suggestion
```
This dep is already specified below.

File: extra-dev/packages/coq-mi-cho-coq/coq-mi-cho-coq.dev/opam [hunk 3]
```diff
@@ -15,10 +15,17 @@ install: [
...
+  "menhir"
+  "coq-menhirlib" {>= "20190626"}
+  "zarith"
+  "ocaml" {>= "4.07.1"}
   "ott" {build & >= "0.29"}
+  "zarith"
```
Comment (palmskog) [inline extra-dev/packages/coq-mi-cho-coq/coq-mi-cho-coq.dev/opam]:
```suggestion
```
Only needs to be specified once.

**Comment by palmskog** [comment]
For reference, the Debian package name appears to be `libgmp-dev` according to `opam-depex`.

**Comment by arvidj** [comment]
Thanks a lot for your suggestions @palmskog !

File: extra-dev/packages/coq-mi-cho-coq/coq-mi-cho-coq.dev/opam [hunk 4]
```diff
@@ -15,9 +15,13 @@ install: [
   make "install"
 ]
 depends: [
-  "ocaml"
-  "coq" {>= "8.8"}
+  "coq-menhirlib" {>= "20190626"}
   "coq-ott" {>= "0.29"}
+  "coq" {>= "8.8"}
+  "menhir"
+  "coq-menhirlib" {>= "20190626"}
```
Comment (palmskog) [inline extra-dev/packages/coq-mi-cho-coq/coq-mi-cho-coq.dev/opam]:
```suggestion
```
This is repeated from above.

**Pull request merged**